Makefile

```diff
@@ -172,7 +173,7 @@ bumpversion-major: ## Bump the version the next major skipping the release
 	bumpversion --no-tag major
 
 CURRENT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
```

Comment:
Can we also add ` 2>/dev/null` to this line?

Reply:
We don't need to redirect to /dev/null, this statement will return the current branch that will be compared later if the current branch is the master branch. We don't need to add it.